frontend/src/features/comparisons/Comparison.tsx

```diff
@@ -61,8 +90,35 @@ const Comparison = () => {
     useState<ComparisonRequest | null>(null);
 
   const searchParams = new URLSearchParams(location.search);
-  const videoA: string = searchParams.get('videoA') || '';
-  const videoB: string = searchParams.get('videoB') || '';
+  const uidParams: { vidA: string; vidB: string } = useMemo(() => {
+    return {
+      vidA: 'uidA',
+      vidB: 'uidB',
+    };
+  }, []);
+  const legacyParams: { vidA: string; vidB: string } = useMemo(() => {
+    return {
+      vidA: 'videoA',
+      vidB: 'videoB',
+    };
+  }, []);
```

Comment:
Instead of using `useMemo`, couldn't these key names be defined as constants outside of the components ?